From 775619f162cdd682289c80daa0cc2efdade0c069 Mon Sep 17 00:00:00 2001
From: Ziyuan Xu <xzy.xu@rock-chips.com>
Date: Fri, 19 Apr 2019 10:27:51 +0800
Subject: [PATCH] arm64: dts: rockchip: mount /root read-write on boot for
 rk3326-linux

fixup mount error on debian:
[    1.990120 ] EXT4-fs (mmcblk1p8): mounted filesystem with ordered
data mode. Opts: (null)
[    1.990932 ] VFS: Mounted root (ext4 filesystem) readonly on device
179:8.

Change-Id: I58cc93c9ce9ba964d75c97298b75a4786812dab9
Signed-off-by: Ziyuan Xu <xzy.xu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10-linux.dts | 2 +-
 arch/arm64/boot/dts/rockchip/rk3326-linux.dtsi            | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10-linux.dts b/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10-linux.dts
index c881254d4396..eec7a7ac0339 100644
--- a/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10-linux.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10-linux.dts
@@ -17,7 +17,7 @@
 	compatible = "rockchip,rk3326-evb-lp3-v10-linux", "rockchip,rk3326";
 
 	chosen {
-                bootargs = "earlycon=uart8250,mmio32,0xff160000 swiotlb=1 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rootfstype=ext4 rootwait";
+                bootargs = "earlycon=uart8250,mmio32,0xff160000 swiotlb=1 console=ttyFIQ0 rw root=PARTUUID=614e0000-0000 rootfstype=ext4 rootwait";
         };
 
 	adc-keys {
diff --git a/arch/arm64/boot/dts/rockchip/rk3326-linux.dtsi b/arch/arm64/boot/dts/rockchip/rk3326-linux.dtsi
index 634a37761f82..3a754df8866c 100644
--- a/arch/arm64/boot/dts/rockchip/rk3326-linux.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3326-linux.dtsi
@@ -44,7 +44,7 @@
 	compatible = "rockchip,linux", "rockchip,rk3326";
 
 	chosen {
-		bootargs = "earlycon=uart8250,mmio32,0xff160000 swiotlb=1 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rootwait";
+		bootargs = "earlycon=uart8250,mmio32,0xff160000 swiotlb=1 console=ttyFIQ0 rw root=PARTUUID=614e0000-0000 rootwait";
 	};
 
 	fiq-debugger {
-- 
2.35.3

